org.eclipse.jdt.launching/launching/org/eclipse/jdt/launching/JavaRuntime.java

```diff
@@ -3337,8 +3331,8 @@ private static void updateCompliance(IVMInstall vm) {
 		if (LaunchingPlugin.isVMLogging()) {
 			LaunchingPlugin.log("Compliance needs an update."); //$NON-NLS-1$
 		}
-        if (vm instanceof IVMInstall2) {
-            String javaVersion = ((IVMInstall2)vm).getJavaVersion();
+		if (vm instanceof IVMInstall2 ivmInstall) {
```

Comment:
intendation looks wrong on github

Reply:
in local there's no modification any intendations. 
<img width="752" alt="image" src="https://github.com/user-attachments/assets/e6f4b259-175f-40ed-b63e-ff4bf7263518">

Reply:
Enable "show whitespace characters" and you'll most likely see a mixture of tabs and spaces and different tools display tabs as 2,4,8,... whitespaces.

Reply:
> Enable "show whitespace characters" and you'll most likely see a mixture of tabs and spaces and different tools display tabs as 2,4,8,... whitespaces.

Thanks